infer_structcol/tests/test_model.py

```diff
@@ -56,12 +58,13 @@ def test_likelihood():
 def test_log_posterior():
     spectrum=Spectrum(500, reflectance = 0.5, sigma_r = 0.1)
     sample = Sample(500, 1.5, 1)
-    theta_range = np.array([[0.35,0.74],[70,160],[1,1000]])
-    
+    theta_range = {'min_phi':0.35, 'max_phi':0.74, 'min_radius':70, 'max_radius': 201, 
+               'min_thickness':1, 'max_thickness':1000}
+        
     # When parameters are within prior range
-    theta1 = (0.5, 150, 200, 0, 0)
+    theta1 = (0.5, 200, 200, 0, 0)
```

Comment:
Why has this changed? What was wrong with the previous values of 160 & 150?
Best practice is to only change tests when there's a good reason to do so.